src/aktualizr_repo/main.cc

```diff
@@ -130,36 +138,47 @@ int main(int argc, char **argv) {
             custom["targetFormat"] = vm["targetformat"].as<std::string>();
           }
           repo.addCustomImage(targetname.string(), *hash, vm["targetlength"].as<uint64_t>(), delegation, custom);
+          std::cout << "Added a target " << targetname.string() << " to the custom image" << std::endl;
```

Comment:
Minor issue, but the custom image IS the target. Might be better to phrase this as "Added a custom image target xyz".